Gemfile

```diff
@@ -9,3 +9,4 @@ end
 gemspec
 
 gem "health_check"
+gem "simple_form", ">= 5.0.1", "< 6.0.0"
```

Comment:
There a Gemfile.lock that needs to be updated?

Reply:
@wadewinningham No, this was already in the Gemfile.lock from the gemspec.